modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenuSmoke.testcase

```diff
@@ -0,0 +1,290 @@
+	@priority = 5
+	test CanBeEnabledByInstanceSettings {
+		task ("When Accessibility Menu is enabled at Instance Settings level") {
+			AccessibilityMenu.activateAccessibilityMenu(portlet = "Instance Settings");
+		}
+
+		task ("Then Accessibility Menu can be reached by keyboard") {
+			AccessibilityMenu.assertCanAccessAccessibilityMenuModal();
+		}
+	}
+
+	@description = "LPS-178192. Verifies that the user can enable it by Site Settings."
+	@priority = 3
+	test CanBeEnabledBySiteSettings {
+		property portal.acceptance = "false";
+
+		task ("When navigate to Site Settings > Accessibility And turn on Enable Accessibility Menu") {
+			AccessibilityMenu.activateAccessibilityMenuBySiteSettings();
+		}
+
+		task ("Then Accessibility Menu can be reached by keyboard") {
+			AccessibilityMenu.assertCanAccessAccessibilityMenuModal();
+		}
+	}
+
+	@description = "LPS-178192.Verifies that the user can enable it by System Settings"
+	@priority = 5
+	test CanBeEnabledBySystemSettings {
+		property test.liferay.virtual.instance = "false";
+
+		task ("When Accessibility Menu is enabled at System Settings level") {
+			AccessibilityMenu.activateAccessibilityMenu(portlet = "System Settings");
+		}
+
+		task ("Then Accessibility Menu can be reached by keyboard") {
+			AccessibilityMenu.assertCanAccessAccessibilityMenuModal();
+		}
+	}
+
+	@description = "LPS-178192 *********Veryfies that settings can be overridden."
+	@priority = 4
+	test CanOverrideSettingsOrderSmoke {
+		property test.liferay.virtual.instance = "false";
+
+		task ("And new virtual instance B is created") {
+			Navigator.openURL();
+
+			HeadlessPortalInstanceAPI.addPortalInstance(
+				domain = "liferay.com",
+				portalInstanceId = "www.instance-b.com",
+				virtualHost = "www.instance-b.com");
+		}
+
+		task ("And new site is created A1") {
+			HeadlessSite.addSite(siteName = "Site A1");
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site A1",
+				layoutName = "Site A1",
+				type = "content");
+
+			JSONLayout.publishLayout(
+				groupName = "Site A1",
+				layoutName = "Site A1");
+		}
+
+		task ("And new site is created A2") {
+			HeadlessSite.addSite(siteName = "Site A2");
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site A2",
+				layoutName = "Site A2",
+				type = "content");
+
+			JSONLayout.publishLayout(
+				groupName = "Site A2",
+				layoutName = "Site A2");
+		}
+
+		task ("When enable Accessibility Menu on site settings A1") {
+			Navigator.gotoSitePage(
+				pageName = "Site A1",
+				siteName = "Site A1");
+
+			AccessibilityMenu.activateAccessibilityMenuBySiteSettings();
+		}
+
+		task ("Then accessibility menu is available on site A1") {
+			AccessibilityMenu.assertCanAccessAccessibilityMenuModal();
+		}
+
+		task ("And accessibility menu is not available on site A2") {
+			Navigator.gotoSitePage(
+				pageName = "Site A2",
+				siteName = "Site A2");
+
+			AccessibilityMenu.assertCannotAccessAccessibilityMenuModal();
```

Comment:
Hey, when I activate the accessibility menu at site settings on `site A1`, it extends to `site A2` while running Poshi... When I try to reproduce it manually, it does not work 😮 ... Do you know what can be done for fixing that?

Reply:
I could solve it... Now I'm stuck at another part 🙅🏻